composer.json

```diff
@@ -13,14 +13,15 @@
         }
     },
     "require": {
-        "php": ">=5.5",
+        "php": "~5.5|~7.0",
```

Comment:
Do you want `~` or `^` here? (again, only wanting clarification that this was intentional)